src/distraction/ListOfDistractions.java

```diff
@@ -0,0 +1,51 @@
+package distraction;
+
+/**
+ *
+ * @author Johnny
+ */
+
+import eu.opends.main.Simulator;
+
+public class ListOfDistractions {
+    
+    private final Simulator sim;
+    private DroppingBox dropBox;
+    private SoundDistraction soundDis;
+    private DarkeningDistraction dark;
+    
+    
+    public ListOfDistractions(Simulator sim){
```

Comment:
do you need this class? AFAIK it randomly spawns a distractor; behavior we want is that if in the settings a distracotr is enabled (and its probability may be set), the simulation will in each step run through the list of instantiated distractors and call a function `bool randomAppear()` (that method can be defined in the super class), if true, spawn the distractor